sshd-core/pom.xml

```diff
@@ -110,6 +110,24 @@
             <artifactId>testcontainers</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-netty-shaded</artifactId>
+            <version>1.27.2</version>
```

Comment:
Let's use a property for the grpc version (D.R.Y.)

Reply:
Done.